terragrunt/modules/crates-io/_terraform.tf

```diff
@@ -91,3 +91,14 @@ variable "static_fastly_weight" {
   description = "Weight of the traffic for static.crates.io that is routed through Fastly"
   type        = number
 }
+
+variable "fastly_customer_id" {
+  description = "Our Customer ID on Fastly"
```

Comment:
Is this something that we want to add as a plain text variable or put into a secret?

Reply:
I *think* we should probably treat it as a secret; any requests coming out of the Fastly AWS account in possession of this ID can assume the role in our account. No one other than Fastly should be issuing such requests of course but no need to expose it IMO.

Reply:
Yeah I feel better if we keep this private as well 👍